Backpack/Tests/SnapshotTests/BPKCardSnapshotTest.m

```diff
@@ -322,6 +322,32 @@ - (void)testDividedSnapshotWithoutPaddedVertical {
     BPKSnapshotVerifyViewDark(darkView, nil);
 }
 
+- (UIView *)createDividedSnapshotWithVerticalSolidLine {
+    UIView *parentView = [[UIView alloc] initWithFrame:CGRectZero];
+    BPKDividedCard *dividedCard = [[BPKDividedCard alloc] initWithPadded:YES];
+    BPKLabel *firstInnerView = [[BPKLabel alloc] initWithFontStyle:BPKFontStyleTextBodyDefault];
+    BPKLabel *secondInnerView = [[BPKLabel alloc] initWithFontStyle:BPKFontStyleTextBodyDefault];
+    firstInnerView.text = @"Lorem ipsum";
+    secondInnerView.text = @"dolor sit amet";
+    firstInnerView.backgroundColor = [BPKColor panjin];
+    secondInnerView.backgroundColor = [BPKColor monteverde];
```

Comment:
Can you please use 
- statusDangerFillColor instead of panjin
- statusSuccessFillColor instead of monteverde

We're in the process of removing deprecated colour tokens from our tests and will take care of the others in this test-class later.